src/common/com/intellij/plugins/haxe/model/HaxeParameterModel.java

```diff
@@ -104,4 +109,20 @@ public String getPresentableText() {
     out += getType().toStringWithoutConstant();
     return out;
   }
+
+  public void remove() {
+    PsiElement psi = getPsi();
+    if (psi != null) {
+      PsiElement prePsi = UsefulPsiTreeUtil.getPrevSiblingSkipWhiteSpaces(psi, true);
+      PsiElement nextPsi = UsefulPsiTreeUtil.getNextSiblingNoSpaces(psi);
+      TextRange range = psi.getTextRange();
+
+      if (prePsi != null && prePsi.getText().equals(",")) {
+        range = range.union(prePsi.getTextRange());
+      } else if (nextPsi != null && nextPsi.getText().equals(",")) {
+        range = range.union(nextPsi.getTextRange());
```

Comment:
Looks like it might leave two commas in place in the case: (existing, toRemove, existing)